testing.cfg

```diff
@@ -6,6 +6,9 @@
     "vesting-addin":       "5tgpCGfXYaZhKWJsrNR4zyp4o4n3wSQ81i5MzPqKEeAK",
     "neon-evm-program":    "DCPSnJHB38e7vNK6o3AVLswJGRaP87iiNx2zvvapiKBz",
     "maintenance-program": "7aPH9mBAvUtJDGV2L1KyvpR5nKF7man5DZzBPaxmisg5",
+    "executables-paths":    ["deploy/evm_loader.so"],
+    "delegates":            ["CVAimMqtcmSUCV4RLZSJAreDpEd7JEZmrvCVj85yaRzZ"],
```

Comment:
Please put the private key into the `artifacts` directory.

Reply:
Fixed